src/settings.rs

```diff
@@ -11,6 +11,9 @@ use crate::channel_value::ChannelValue;
 /// Default [`nix_package_url`](CommonSettings::nix_package_url) for Linux x86_64
 pub const NIX_X64_64_LINUX_URL: &str =
     "https://releases.nixos.org/nix/nix-2.12.0/nix-2.12.0-x86_64-linux.tar.xz";
```

Comment:
Maybe not in this PR but:

Should we be bumping the Nix version? Latest stable is already 2.13.2.

Reply:
Ya! I was bumping it in #196 but that is delayed so I can do that separately.